src/pages/today/index.tsx

```diff
@@ -25,32 +58,63 @@ export type HabitTodo = {
 export default function Today() {
 	const [date, setDate] = useState(new Date());
 	const [habitTodos, setHabitTodos] = useState<HabitTodo[]>([]);
+	const documentId = useRef<string>(null);
 
 	const completedHabits = habitTodos.filter((habit) => habit.isComplete);
 	const incompleteHabits = habitTodos.filter((habit) => !habit.isComplete);
 
 	const isDateToday = isToday(date);
 	const isDateYesterday = isYesterday(date);
 
+	const handleCheckHabit =
+		(habitId: string) => (e: React.ChangeEvent<HTMLInputElement>) =>
```

Comment:
Minor, but instead of `React.ChangeEvent` you could just import ChangeEvent from React on line 38